### ISSUE TITLE
img tags should have descriptive alt text

### ISSUE THREAD

**Issue by discospiff** [beyrer/IS7012#11]
For accessibility, images should have descriptive alt text.  That is, it should describe what's in the image for someone who can't see the image.  Don't say, "A picture of me".  Instead, say something like, "I'm a 6' tall redhead, wearing a dark suit and green tie, in front of a white wall".

https://github.com/beyrer/IS7012/blob/67c66d3f6a32c4ec261f7cf2d4b02aaded4266cb/BearcatSportsBuyback/Pages/Index.cshtml#L81
